.gitignore

```diff
@@ -11,3 +11,6 @@ build/
 #*
 *.html
 *.log
+*.journal
+citest.egg-info/
+venv/
```

Comment:
citest.egg-info is a resulting file from `pip install -r requirements.txt` - it's where the python package info is kept. These are for ease of development, before I couldn't rebase properly without clearing my local `venv` state.